external/fv3fit/fv3fit/pytorch/graph/train.py

```diff
@@ -4,7 +4,7 @@
 from fv3fit._shared.training_config import Hyperparameters
 from toolz.functoolz import curry
 from fv3fit.pytorch.predict import PytorchModel
-from fv3fit.pytorch.graph.network import GraphNetwork, GraphNetworkConfig
+from fv3fit.pytorch.graph.UNet_GraphNet import GraphUNet, UNetGraphNetworkConfig
```

Comment:
Please rename the module file to something snake_case and descriptive, not sure that "net" needs to be in the name twice. Maybe even just `unet` since this is already under the `graph` namespace?